recipes/esys-escript/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -x -e
+set -o pipefail
+
+CXXFLAGS="${CXXFLAGS} -fPIC -w -fopenmp"
+
+cd ${SRC_DIR}/escript
+scons -j"${CPU_COUNT}" \
+    options_file="${SRC_DIR}/escript/scons/templates/stretch_options.py" \
+    prefix=${PREFIX} \
+    build_dir=${BUILD_PREFIX}/escript_build \
+    boost_prefix=${PREFIX} \
+    boost_libs="boost_python27" \
+    cxx=${CXX} \
+    cxx_extra="-w -fPIC" \
+    ld_extra="-L${LIBRARY_PATH} -lgomp -lomp" \
```

Comment:
Use just `-fopenmp` instead of specifying the library like `-lgomp -lomp`

Reply:
I have implemented these changes.

Reply:
If `-fopenmp` doesn't work. Use `-lgomp` for linux and `-lomp` for osx

Reply:
@aellery, you need to fix this. `-lgomp` is for linux and `-lomp` is for osx

Reply:
I've removed the `-lomp` flag.